docs/sequence-diagrams/user-service.puml

```diff
@@ -0,0 +1,16 @@
+@startuml
+Client -> Frontend: Register as new user
+Frontend -> User: POST /api/user/signup
+User -> Frontend: Success
+Frontend -> Client: Redirect to login
+Client -> Frontend: Login
+Frontend -> User: POST /api/user/login
```

Comment:
Update route from 
/api/user/signup to /signup
and 
/api/user/login to /login

Reply:
Okay